spec/system/user_logs_in_with_password_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe "Authenticating with a password" do
```

Comment:
I think we had no specs for password logins so far, so I added a whole bunch, not only the ones for account locking.

Reply:
Nice!🎉